src/pages/developers/tutorials.tsx

```diff
@@ -80,6 +88,7 @@ const Author = styled.p`
   text-transform: uppercase;
 `
 
+// Todo: Reformat this to not use type assertions (this is a workaround to avoid any and keep compiler happy)
 const Title = styled.p`
```

Comment:
Lets define the custom props here.
```suggestion
const Title = styled.p<{ isExternal: boolean }>`
```